Microsoft.DotNet.Interactive/Kernel.cs

```diff
@@ -21,9 +21,14 @@ public static class Kernel
         {
             var displayId = DisplayIdGenerator?.Invoke() ?? Guid.NewGuid().ToString();
 
-            if (value is string)
+            switch (value)
```

Comment:
This also should move to someplace in the formatter infrastructure, where people can register override mime types for specific types.